examples/file_operation/example_file_operation.cpp

```diff
@@ -0,0 +1,63 @@
+// Example application of library utilization for bitmaps
+#include <iostream>
+#include "../../src/image_buffer.h"
+#include "../../src/image_function.h"
+#include "../../src/file/file_image.h"
+
+int main( int argc, char * argv[] )
+{
+    std::vector<std::string> filePaths;
+
+    if ( argc > 1 ) // Check input data
+    {
+        for ( int i = 1; i < argc; ++i)
+            filePaths.push_back(argv[i]);
+    }
+
+    else
+    {
+        filePaths.push_back("../../data/lena.bmp");
+        filePaths.push_back("../../data/ape.png");
+//        filePaths.push_back("../../data/snail.jpg");
+    }
+
+
+    try // <---- do not forget to put your code into try.. catch block!
+    {
+            // Load an image from storage
+        for ( auto it = filePaths.begin(); it != filePaths.end(); ++it )
```

Comment:
Agree about auto. But isn't more native to use iterators like vector<string>::iterator explicitly instead of indexing? or it's longer and less understandable?

Reply:
Let's use indexing instead like `filePaths[i]`. We need to think from the point of view that not all developers are familiar with iterators :) Examples always must be user-friendly and very easy to understand.